safe_cli/prompt_parser.py

```diff
@@ -165,7 +165,7 @@ def load_cli_owners(args):
 
     @safe_exception
     def load_ledger_cli_owners(args):
-        safe_operator.load_ledger_cli_owners(args.legacy_accounts)
+        safe_operator.load_ledger_cli_owners(args.derivation_path, args.legacy_accounts)
```

Comment:
As you are passing optional arguments, it's recommended to add them when calling the function